foxglove/console/api.go

```diff
@@ -145,6 +145,90 @@ func (r AttachmentsResponse) Headers() []string {
 	}
 }
 
+type RecordingsRequest struct {
+	DeviceID     string `json:"device.id" form:"device.id,omitempty"`
+	Start        string `json:"start" form:"start,omitempty"`
+	End          string `json:"end" form:"end,omitempty"`
+	Path         string `json:"path" form:"path,omitempty"`
+	SiteID       string `json:"siteId" form:"siteId,omitempty"`
```

Comment:
why are these camelcase?
```suggestion
	SiteID       string `json:"site.id" form:"site.id,omitempty"`
```